payments/views.py

```diff
@@ -103,6 +105,9 @@ def pay(request, order):
             order=order.pk, err=payment.error))
         messages.error(request, _('Something went wrong, try again later.'))
 
+        order.processing_fee -= paypal_fee
```

Comment:
we end up here only if there is problem with PayPal, otherwise he is redirected to PayPal and this code is never executed. If PayPal has error, than paypal_fee no longer applies as user have only chance to pay by bank transfer (or other method we implement in future)